common/build.gradle

```diff
@@ -22,5 +22,7 @@ if (tasks.findByName('trimShadedJar')) {
         keep "class com.linecorp.centraldogma.internal.shaded.caffeine.** { *; }"
         // Prevent ProGuard from removing all enum values from Option because otherwise it becomes a non-enum class.
         keep "class com.linecorp.centraldogma.internal.shaded.jsonpath.Option { *; }"
+
+        dontnote
```

Comment:
This reduces the verbosity of ProGuardTask when running in parallel.
The original comment: https://github.com/line/armeria/pull/2172/files#r332367951

Reply:
Please add a line comment about it.